packages/cli/package.json

```diff
@@ -64,7 +64,7 @@
     "envinfo": "^7.8.1",
     "esm": "^3.2.25",
     "file-loader": "^6.2.0",
-    "fork-ts-checker-webpack-plugin": "^4.0.4",
+    "fork-ts-checker-webpack-plugin": "^6.5.3",
```

Comment:
There should likely be a change in the repo's lock file to accompany this.

Reply:
> There should likely be a change in the repo's lock file to accompany this.

uhm, are you referring to yarn.lock? in such case, my bad! 😄 I only use npm and I totally missed it! Could you briefly tell me how such file it's supposed to be updated and I will look into it as well. (it looks some sort of autogenerated file and not just a matter of updating the version like it's for package.json)

Reply:
Just run `yarn` / `yarn install`. It's the same as a `package-lock.json`.